src/config/config.ts

```diff
@@ -402,26 +242,25 @@ const otpGenerator = () => {
 }
 
 const config: Config = {
-  app: appConfig,
+  app: basicVars.appConfig,
   db: dbConfig,
   aws: awsConfig,
   mail: mailConfig,
   cookieSettings,
   isDev,
   nodeEnv,
-  port,
-  customCloudWatchGroup,
-  sessionSecret,
-  otpLifeSpan,
-  bounceLifeSpan,
-  formsgSdkMode,
-  chromiumBin,
-  cspReportUri,
-  submissionsTopUp,
-  isGeneralMaintenance,
-  isLoginBanner,
-  siteBannerContent,
-  adminBannerContent,
+  formsgSdkMode: basicVars.formsgSdkMode,
+  customCloudWatchGroup: basicVars.awsConfig.customCloudWatchGroup,
+  bounceLifeSpan: basicVars.mail.bounceLifeSpan,
+  chromiumBin: basicVars.mail.chromiumBin,
+  port: basicVars.core.port,
+  sessionSecret: basicVars.core.sessionSecret,
+  otpLifeSpan: basicVars.core.otpLifeSpan,
+  submissionsTopUp: basicVars.core.submissionsTopUp,
+  isGeneralMaintenance: basicVars.banner.isGeneralMaintenance,
+  isLoginBanner: basicVars.banner.isLoginBanner,
+  siteBannerContent: basicVars.banner.siteBannerContent,
+  adminBannerContent: basicVars.banner.adminBannerContent,
```

Comment:
again, `cspReportUri` is missing here, was that on purpose?

Reply:
yeap on purpose. Good catch. Moved it to the bonus feature ~ sentry